tests/unit/changeDir.ts

```diff
@@ -0,0 +1,22 @@
+import * as registerSuite from 'intern!object';
+import * as assert from 'intern/chai!assert';
+import changeDir from 'src/changeDir';
+import { stub, SinonStub } from 'sinon';
+
+let changeDirStub: SinonStub;
+
+registerSuite({
+	name: 'changeDir',
+	'setup'() {
+		changeDirStub = stub(process, 'chdir');
```

Comment:
You could probably have a single module that exports all the things that you need to stub (like `__dirname`)